R/EXADBI-info.R

```diff
@@ -0,0 +1,80 @@
+#' @include EXADBI-object.R
+#' @include EXADBI-driver.R
+#' @include EXADBI-result.R
+#' @include EXADBI-connection.R
+NULL
+
+#--------------------- dbGetInfo --------------------------
+
+#' @title dbGetInfo
+#' @describeIn dbGetInfo Returns metadata on a given EXAObject.
+#'
+#' @param dbObj An EXAObject.
+#' @return A named list.
+#' @export
+#' @author EXASOL AG <opensource@exasol.com>
```

Comment:
Is it a requirement to set author to every exported method?

Reply:
The @author key was set on some methods, but not all, so I found it better to add it on all methods.

Reply:
Got it :+1: